src/utils/variables.ts

```diff
@@ -35,18 +25,19 @@ export const formatVariables = (list: any[]) => {
   return variables;
 };
 
-export const decomposeVariables = (variables: Variables) => {
+export const decomposeVariables = (variables: Variables): VariableInForm[] | [] => {
```

Comment:
`[]` will match any expected array.

```suggestion
export const decomposeVariables = (variables: Variables): VariableInForm[] => {
```